plotly/plotly/plotly.py

```diff
@@ -1520,6 +1520,77 @@ def get_dashboard_names(cls):
         return [str(dboard['filename']) for dboard in dashboards]
 
 
+class presentation_ops:
+    """
+    Interface to Plotly's Spectacle-Presentations API.
+    """
+    @classmethod
+    def upload(cls, presentation, filename, sharing='public', auto_open=True):
+        """
+        Function for uploading presentations to Plotly.
+
+        :param (dict) presentation: the JSON presentation to be uploaded. Use
+            plotly.presentation_objs.Presentation to create the presentation
+            from a string.
```

Comment:
I don't quite understand what "from a string" means here

Reply:
What I mean is that you are defining a variable as a string and passing that into the presentations function to generate the JSON

Reply:
OK. Maybe "... to create presentations from a Markdown-like string"?